### PR TITLE
Package updates 2021-03-29

### PR DESCRIPTION
**Pull request by znicholasbrown**
PR Checklist:

- [x] add a short description of what's changed to the top of the `CHANGELOG.md`
- [ ] add/update tests (or don't, for reasons explained below)

## Describe this PR
Updates all non-pinned packages, removes some unused packages related to auth, and updates all deprecated typography-related classes.

### CODE REVIEW

File: src/components/ChartCard/ChartCardHeader-Ring.vue [hunk 1]
```diff
@@ -41,7 +41,7 @@ export default {
     mainClasses() {
       return {
         'display-3': this.$vuetify.breakpoint.lgAndUp,
```
Comment (whimsicallyson) [inline src/components/ChartCard/ChartCardHeader-Ring.vue:43]: vuetify's still ok with this one?

Comment (znicholasbrown) [reply src/components/ChartCard/ChartCardHeader-Ring.vue:43]: Hm, probably doesn't know it exists - I'll update it

Comment (znicholasbrown) [reply src/components/ChartCard/ChartCardHeader-Ring.vue]: Updated 👍

**Review by whimsicallyson — APPROVED** [review]
all makes sense to me

**Pull request merged**